release/release.go

```diff
@@ -105,15 +103,14 @@ func readOSRelease() OS {
 	return osRelease
 }
 
-var ioutilReadFile = ioutil.ReadFile
+// Note that osutil.FileExists cannot be used here as an osutil import will create a cyclic import
+var fileExists = func(path string) bool {
```

Comment:
(nitpick^9) The mocking could be avoided entirely if the code below would use `if osutil.FileExists(filepath.Join(dirs.GlobalRootDir, "/proc...")` - then the tests could just do a `tmpdir := c.MkDir() ; dirs.SetRootDir(tmpdir)` and just create the file inside this tmp env but as it is it's totally fine :)

Reply:
You're right that it would look cleaner :) However, importing `osutil` would cause a dependency loop.

Reply:
Heh, I have a deja-vu right now :) Sorry!